api/python/cellxgene_census/pyproject.toml

```diff
@@ -31,7 +31,7 @@ dependencies= [
     # NOTE: the tiledbsoma version must be >= to the version used in the Census builder, to
     # ensure that the assets are readable (tiledbsoma supports backward compatible reading).
     # Make sure this version does not fall behind the builder's tiledbsoma version.
-    "tiledbsoma==1.2.2",
+    "tiledbsoma==1.2.3",
```

Comment:
given this update, please remove the `tiledb` pin below, and also update the PR description to note that this PR will _also_ fix #466

Reply:
Also, is there an equivalent update required for the R build?

Reply:
Removed. R has a similar upgrade needed but it's not straightforward. I'll do that as a separate PR.